Assets/Scripts/State/IdleState.cs

```diff
@@ -26,11 +29,41 @@ public IdleState(Character character, State nextState = null)
 
         public override void Update(float deltaTime)
         {
+                Tile startTile = character.CurrTile;
+                Tile endTile = null;
+                List<Tile> path = null;
+
+                // Get a random Int 0,1,2,3 for representing the 4 directions.
+                int direction = Int32.Parse(Math.Truncate(Random.value * 4).ToString());
+
+                switch (direction)
+                {
+                    case 0:
+                        endTile = World.Current.GetTileAt(startTile.X + 1, startTile.Y, startTile.Z);
+                        break;
+                    case 1:
+                        endTile = World.Current.GetTileAt(startTile.X, startTile.Y + 1, startTile.Z);
+                        break;
+                    case 2:
+                        endTile = World.Current.GetTileAt(startTile.X - 1, startTile.Y, startTile.Z);
+                        break;
+                    case 3:
+                        endTile = World.Current.GetTileAt(startTile.X, startTile.Y - 1, startTile.Z);
+                        break;
+                }
+
```

Comment:
Truncate is unnecessary with Random.Range, if both parameters are ints, it should return an int.

Also for your use, when doing code, if you do 3 backticks, a line break, then the code, end with a line break and another 3 backticks and it will properly include the linebreaks in your code.